### PR TITLE
add script for rotating cloud.gov secrets and runbook for description of script

### PR DESCRIPTION
**Pull request by loganmeetsworld**
# Setup Github Cloud.gov CD Secrets Rotate  #

## 🗣 Description ##

This adds a script and documentation for the process of rotating our cloud.gov secrets, which is something we must routinely do but is also important to have a runbook for rotating if the secrets are compromised.

You can see the repository secrets successfully get set here: 

Screen Shot 2022-08-12 at 9 22 49 AM

This will not be used until we setup a github action. We can decide at that point which environments we'd like to CD to and what criteria we have for deployment. 

Only people with admin access (SpaceDevelopers) will be able to run this script.

### CODE REVIEW

**Comment by loganmeetsworld** [comment]
Part of #11

File: scripts/rotate_cloud_secrets.sh
Comment (SSPJ) [inline scripts/rotate_cloud_secrets.sh:3]: Would suggest making this line more portable, since future maintainers may not have brew on their OS.

```suggestion
if [ ! $(command -v gh) ] || [ ! $(command -v jq) ]; then
```

File: scripts/rotate_cloud_secrets.sh
Comment (SSPJ) [inline scripts/rotate_cloud_secrets.sh:42]: I had to change this `jq` command to get it to work. I figured it was a typo in the PR but could you double check on your end?

This is what the output of `cf` looks like for me:

```json

{
  "credentials": {
    "password": "hhhhhhhhhh",
    "username": "hhhhhhhhhh"
  }
}

```
```suggestion
cf service-key github-cd-account github-cd-key | sed 1,2d  | jq -r '.credentials|[.username, .password]|@tsv' | 
```

**Review by SSPJ — APPROVED** [review]
Nice work! 👍 

I had one issue that prevented the script running and an additional suggestion for interoperability. Good to merge once feedback is addressed.

File: scripts/rotate_cloud_secrets.sh
Comment (loganmeetsworld) [reply scripts/rotate_cloud_secrets.sh:40]: Ahhh this might be a difference in versions of `cf`!

Comment (loganmeetsworld) [reply scripts/rotate_cloud_secrets.sh:40]: What version of cf are you running?

Comment (SSPJ) [reply scripts/rotate_cloud_secrets.sh:40]: That was it! The version at 4d44c86 works fine when I use `cf version 7.5.0+0ad1d63.2022-06-04`.

Since this script will break on future version of `cf`, I'd recommend indicating the fix in a code comment to speed up future debugging.

**Pull request merged**
